jcloud/api.py

```diff
@@ -305,3 +305,10 @@ def survey(args):
     from .survey import Survey
 
     Survey().ask(-1)
+
+
+@asyncify
+async def update(args):
+    from rich import print
+    print(f'Updating Flow: [green]{args.flow_id}[/green]')
```

Comment:
Please avoid `print` here and rely on `logger` inside the `update()` method, so users of `CloudFlow` can disable logs.

Reply:
This is not update method of the CloudFlow. It is api that creates a CloudFlow instance and calls the CloudFlow.update() method.